spec/jobs/cleanup_account_job_spec.rb

```diff
@@ -6,12 +6,7 @@
   end
 
   before do
-    # stub switch so we don't need to worry about changing resource handles
-    allow(account).to receive(:switch) do |&block|
-      block.call
-    end
-
-    allow(RemoveSolrCollectionJob).to receive(:perform_later)
```

Comment:
It looks like I accidentally deleted too much. Thanks for the note. Will get that reinstated (and fix the Travis test failure here)